src/renderer/vsync/vsync_macos.rs

```diff
@@ -77,22 +80,21 @@ impl VSyncMacos {
                 error!("Failed to create display link, CVReturn code: {}.", code);
                 None
             }
-        }
+        };
+        self.display_link = display_link;
     }
 
-    pub fn wait_for_vsync(&mut self) {
-        let (lock, cvar) = &*self.vsync_count;
-        let count = cvar
-            .wait_while(lock.lock().unwrap(), |count| *count < self.last_vsync + 1)
-            .unwrap();
-        self.last_vsync = *count;
+    pub fn wait_for_vsync(&mut self) {}
+
+    pub fn request_redraw(&mut self) {
+        self.redraw_requested.store(true, Ordering::Relaxed);
```

Comment:
Shouldn't this be `Ordering::Acquire` and the `swap` above `Ordering::AcqRel`, as this is relying on the `redraw_requested` to be propagated before it's seen above?

Reply:
Acquire and Release are only really needed if there are some other memory involved, but there we are just comparing a boolean value.

Reply:
Alright!